packages/serverless-orchestration/src/ServerlessSpoke.js

```diff
@@ -105,18 +124,19 @@ function _execShellCommand(cmd, inputEnv, strategyRunnerSpoke = false) {
 // Format stdout outputs. Turns all logs generated while running the script into an array of Json objects.
 function _stripExecStdout(output, strategyRunnerSpoke = false) {
   if (!output) return output;
-  // Parse the outputs into a json object to get an array of logs. It is possible that the output is not in a parable form
-  // if the spoke was running a process that did not correctly generate a winston log. In this case simply return the stripped output.
+  // Parse the outputs into a json object to get an array of logs. It is possible that the output is not in a parable
+  // form if the spoke was running a process that did not correctly generate a winston log. In this case simply return
+  // the stripped output. Note that we use an array to preserve the log ordering.
+  const strippedOutput = _regexStrip(output).replace(/\r?\n|\r/g, ","); // Remove escaped new line chars. Replace with comma between each log output.
```

Comment:
Not sure if you noticed or if this was intentional, but the stripped output in this line is different than the one originally in the catch -- the catch replaced with a space, but the try replaced with a `,`, so this is _technically_ a functionality change. Probably doesn't matter as I can't understand the original intention behind the difference